frontend/src/index.js

```diff
@@ -4,6 +4,8 @@ import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import App from './App';
```

Comment:
are the changes in this file made for `AlertConfiguration` to run locally without needing the login?

Reply:
I think so. I added a route to AlertConfiguration to the NavBar so that I can run locally